### PR TITLE
Extended shift-v items menu

### PR DESCRIPTION
**Pull request by 8street**
#### Summary
```SUMMARY: Interface "Extended shift-v items menu"```

#### Purpose of change
If a player is looking around (shift-v items menu), then now full and not cropped item names are displayed.

#### Describe the solution
I'd added cycle which search max items length and resize shift-v items window. Also there is check for max game window width.

#### Describe alternatives you've considered
Scrolling, but all the appropriate keys are busy.

#### Testing
fix9

### CODE REVIEW

**Comment by KorGgenT** [comment]
how does this look with minimum screen size

File: src/game.cpp [hunk 1]
```diff
@@ -7184,8 +7184,21 @@ void game::list_items_monsters()
 
 game::vmenu_ret game::list_items( const std::vector<map_item_stack> &item_list )
 {
+    std::vector<map_item_stack> ground_items = item_list;
     int iInfoHeight = std::min( 25, TERMY / 2 );
-    const int width = 44;
+    int width = 44;
+
+    //find max length of item name and resize window width
+    for( const map_item_stack cur_item : ground_items ) {
```
Comment (BevapDin) [inline src/game.cpp]:
```suggestion
    for( const map_item_stack &cur_item : ground_items ) {
```
There is really no need to copy the object.

**Comment by 8street** [comment]
> how does this look with minimum screen size
fix10

Umm, I figured out how to modify this PR. Coming soon.

**Comment by 8street** [comment]
fix11

Now it looks more adequate with a minimum window size. With a large window, it looks like in my first screenshot.

File: src/game.cpp (hunk 1 above)
Comment (8street) [reply src/game.cpp]: Done. Thanks.

**Comment by 8street** [comment]
I'd resolved conflicts. Can be merged. Also PR fixes #36787: width of the list_items was also hardcoded in `reset_item_list_state()`, with code `const int width = 44;`, which did not change there.

**Pull request merged**